doc/source/index.rst

```diff
@@ -0,0 +1,333 @@
+.. pytroll-collectors documentation master file, created by
+   sphinx-quickstart on Tue Dec 22 16:13:24 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to pytroll-collectors's documentation!
+==============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+Pytroll-collectors is a collection of scripts and modules to aid in
+operational processing of satellite reception data.
+The scripts run continuously (like daemons) and
+communicate with each other using `posttroll`_ messages.
+
+For example, a chain for processing MetOp AVHRR data from direct
+reception, in which external software deposits files on the file system,
+may look somewhat like this:
+
+* The chain starts with :ref:`trollstalker` to monitor files.
+  :ref:`trollstalker` uses inotify and sends a `posttroll`_ message
+  when a file appears.
+* This message is received by :ref:`gatherer`.  Depending on the
+  reception system, a single MetOp AVHRR overpass may produce multiple files.
+  :ref:`gatherer` determines what files belong together in a region and sends
+  a posttroll message containing all those filenames.
+* AVHRR data need preprocessing with the external software `AAPP`_ before
+  `Satpy`_ can read the data.  This preprocessing can be done with
+  `aapp-runner`_, For this preprocessing, it is advantageous to pass a single
+  file.  Therefore, the :ref:`cat` script may be listening to messages from
+  :ref:`gatherer` and concatenate files together (it will need `Kai`_ to do
+  so).  When done, it sends another message.
+* For pre-processing data with AAPP and ANA, `aapp-runner`_ is responsible
+  and can be configured to read posttroll messages either from :ref:`cat` or
+  directly from :ref:`gatherer`.  See documentation for `aapp-runner`_.
+
+The exact configuration depends on factors that will vary depending on
+what satellite data are processed, whether thore are from direct readout,
```

Comment:
"thore"? I am not a native english speaker... but haven't used/seen that before?

Reply:
Fixed.